app/views/responsible_body/internet/mobile/manual_requests/confirm.html.erb

```diff
@@ -14,9 +14,9 @@
           <%= @extra_mobile_data_request.account_holder_name %>
         </dd>
         <dd class="govuk-summary-list__actions">
-          <%= govuk_link_to('', new_responsible_body_internet_mobile_manual_request_path( anchor: 'extra-mobile-data-request-account-holder-name-field') ) do %>
-            Change<span class="govuk-visually-hidden"> Account holder name</span>
-          <%- end %>
+          <%= govuk_link_to('Change<span class="govuk-visually-hidden"> Account holder name</span>'.html_safe,
```

Comment:
(We should push the block logic upstream to govuk-components, so that `govuk_link_to` continues to act like the native `link_to` does.)

Reply:
Good point, I'll take that up with govuk-components but I don't see that being a blocker for us to get this merged.